data-reconciliation-app/docs/adr/03-reconciliation-logic.md

```diff
@@ -11,37 +11,37 @@ Once data is ingested into the app, Members/Users will query for the data to be
 ## Requirements
 
 - Reconciliation is on each record.
```

Comment:
Do we ever define what "Reconciliation" is?

Reply:
Added definition